test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -325,29 +343,49 @@ std::string RandomCode::fillArguments(dev::eth::Instruction _opcode, RandomCodeO
 
 		switch (_opcode)
 		{
-		case dev::eth::Instruction::CREATE:
+		case eth::Instruction::EXTCODECOPY:
+			code += getPushCode(randUniIntGen() % 32);  //memstart2
```

Comment:
Random memory address is generated here.
What I'm asking is make it look like `code += getPushCode(randMemAddresstGen());  //memstart2`